### PR TITLE
Newton fake ui

### PR DESCRIPTION
**Pull request by rkhozinov**

### CODE REVIEW

File: dev-server/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+mkdir -p $ARTIFACTS
+
+export NAILGUN_STATIC=$ARTIFACTS/static
+
+GULP="./node_modules/.bin/gulp"
+TESTS_DIR=static/tests/functional
+TESTS=$TESTS_DIR/test_*.js
+
+#echo "Building UI..."
+#${GULP} build --no-sourcemaps --extra-entries=sinon --static-dir=$NAILGUN_STATIC
```
Comment (ekhomyakova) [inline dev-server/entrypoint.sh]: Remove

File: docker-compose.yml [hunk 2]
```diff
@@ -1,26 +1,86 @@
...
+      args:
+        - FUEL_WEB_REQS=${FUEL_WEB_REQS}
+    volumes:
+      - ./dev-server:/dev-server
+      - ./fuel-ui:/fuel-ui
+      - ./fuel-web:/fuel-web
+      - ./dev-server/log:/var/log/nailgun
+    entrypoint: /dev-server/entrypoint.sh
     ports:
       - "9057"
```
Comment (ekhomyakova) [inline docker-compose.yml]: Is it needed?

Comment (rkhozinov) [inline docker-compose.yml]: remove that

File: base/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```
Comment (rkhozinov) [inline base/entrypoint.sh:1]: remove

File: docker-compose.yml [hunk 4]
```diff
@@ -15,11 +15,42 @@ services:
...
+      - .env
+    build:
+      context: dev-server
+      args:
+        - TZ=Europe/Czech
+    volumes:
+      - ./dev-server:/dev-server
+      - ./fuel-ui:/fuel-ui
+      - ./fuel-web:/fuel-web
+      - ./dev-server/log:/var/log/nailgun
```
Comment (rkhozinov) [inline docker-compose.yml]: remove that

File: docker-compose.yml [hunk 5]
```diff
@@ -15,11 +15,42 @@ services:
...
+    entrypoint: /dev-server/entrypoint.sh
+    environment:
       - NAILGUN_HOST=${NAILGUN_HOST}
```
Comment (rkhozinov) [inline docker-compose.yml]: remove

**Pull request merged**